lib/bundler/graph.rb

```diff
@@ -135,7 +135,7 @@ def run
         @groups.each do |group|
           g.add_nodes(
             group,
-            { :style     => "filled",
+            { :style => "filled",
```

Comment:
this makes no sense to me... why is the first line not allowed to be aligned, but the rest are?